src/qt/walletcontroller.cpp

```diff
@@ -287,7 +287,7 @@ void OpenWalletActivity::open(const std::string& path)
 {
     QString name = path.empty() ? QString("["+tr("default wallet")+"]") : QString::fromStdString(path);
 
-    showProgressDialog(tr("Opening Wallet <b>%1</b>...").arg(name.toHtmlEscaped()));
+    showProgressDialog(tr("<div align='center'>Opening Wallet</div><hr><div align='center'><b>%1</b></div>").arg(name.toHtmlEscaped()));
```

Comment:
Duplicated code could be moved into the `showProgressDialog()` body, no?

Reply:
html code should not be translated